share/ci/install/pypicongpu.sh

```diff
@@ -40,7 +40,7 @@ conda create -n pypicongpu python=${PYTHON_VERSION}
 conda activate pypicongpu
 python3 --version
 MODIFIED_REQUIREMENT_TXT=$CI_PROJECT_DIR/lib/python/picongpu/modified_requirements.txt
-python3 $CI_PROJECT_DIR/share/ci/install/requirements_txt_modifier.py $CI_PROJECT_DIR/lib/python/picongpu/requirements.txt $MODIFIED_REQUIREMENT_TXT
+python3 $CI_PROJECT_DIR/share/ci/install/requirements_txt_modifier.py $CI_PROJECT_DIR/lib/python/picongpu/picmi/requirements.txt $MODIFIED_REQUIREMENT_TXT
```

Comment:
I am not sure about this one

Reply:
It is tested in the `pypicongpu-full-matrix` test. The dependencies are installed.